src/vm/ecalllist.h

```diff
@@ -958,8 +958,8 @@ FCFuncStart(gAppDomainFuncs)
 #ifndef FEATURE_CORECLR
     FCFuncElement("GetSecurityDescriptor", AppDomainNative::GetSecurityDescriptor)
     FCFuncElement("nIsDefaultAppDomainForEvidence", AppDomainNative::IsDefaultAppDomainForEvidence)
-    FCFuncElement("nGetAssemblies", AppDomainNative::GetAssemblies)
 #endif
+    FCFuncElement("nGetAssemblies", AppDomainNative::GetAssemblies)
```

Comment:
Yes, this need to be plumbed via AssemblyLoadContext. @rahku Please skip adding GetAssemblies - I will take care of it - and you can proceed with UnhandledException handler.